docs/home/contribute/style-guide.md

```diff
@@ -15,6 +15,8 @@ docs, follow the instructions on
 
 {% capture body %}
 
+**Note:** Kubernetes documentation uses [GitHub Flavored Markdown](https://github.github.com/gfm/).
```

Comment:
add ` {: .note}` in the next line, please.

Reply:
Just out of curiosity why we add {: .note} this. I have seen this in many places but always wondered why it is used.

Reply:
:) there is a preview you can check:

https://deploy-preview-6229--kubernetes-io-master-staging.netlify.com/docs/home/contribute/style-guide/